### DISCUSSION TITLE
Guest / Temporary profile

### DISCUSSION THREAD

**Discussion by kmaks-dev** [cefsharp/CefSharp#3435] [Q&A]
I'm trying to launch separatate CEFSharp forms with each having a separate profile. Currently if i log in to Azure AD panel the same credential is used in Sharepoint Online or Exchange Online.

I can reach the desired functionality if my program starts "chrome.exe -guest https://*" but can't reach the same functionality with CEFSharp forms. 

Can anyone point me into the right direction?

**Comment by amaitland** [comment]
See https://github.com/cefsharp/CefSharp/wiki/General-Usage#request-context-browser-isolation
